templates/rhpam73-managed.yaml

```diff
@@ -41,10 +41,15 @@ parameters:
   description: Maven mirror to use by Business Central and KIE server.
   name: MAVEN_MIRROR_URL
   required: false
+- displayName: Maven mirror of
+  description: Maven mirror configuration for KIE server.
+  name: MAVEN_MIRROR_OF
+  value: "external:*,!repo-rhpamcentr"
```

Comment:
We wanted to default this to "external:*,!repo-custom" because a custom Maven repo is expected to be used, while repo-rhpamcentr is normally NOT present in prod and managed?